### PR TITLE
styles(Accordion): added focus ring

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

It adds focus ring for `Accordion` trigger

Screenshot 2022-10-04 at 14 05 59

### CODE REVIEW

**Comment by gustaveen** [comment]
Nice! 🙌 I wonder how we could style this to harmonise with the design. It almost looks like a checkbox with a square around it :) Because this would show up on click as well right?

**Comment by guilhermespopolin** [comment]
> Nice! 🙌 I wonder how we could style this to harmonise with the design. It almost looks like a checkbox with a square around it :) Because this would show up on click as well right?
That's true, it does look like a checkbox 😅
It would only appear by focusing via keyboard tho

File: apps/store/src/components/PriceForm/Accordion.tsx [hunk 1]
```diff
@@ -22,6 +22,13 @@ export const Content = styled(Accordion.Content)(({ theme }) => ({
   paddingTop: theme.space[0],
 }))
 
+const StyledTrigger = styled(Accordion.Trigger)(({ theme }) => ({
+  '&:focus-visible': {
+    outline: `2px solid ${theme.colors.gray900}`,
```
Comment (alebedev) [inline apps/store/src/components/PriceForm/Accordion.tsx:27]: Let's ask design team, maybe we want to use some highlight color here

Comment (guilhermespopolin) [reply apps/store/src/components/PriceForm/Accordion.tsx:27]: So it seems any designs in that direction is [gonna take some time to get done](https://hedviginsurance.slack.com/archives/C041SD67G82/p1664895741834449). Should we close this PR them? 🤔

Comment (simonauner) [reply apps/store/src/components/PriceForm/Accordion.tsx:27]: I would argue the counterpart @guilhermespopolin . Merge first and then we can tweak later. Accessibility is a feature 🚀

**Pull request merged**